src/com/chschmid/jdotxt/gui/controls/JdotxtTaskPanel.java

```diff
@@ -43,6 +45,16 @@
 
 @SuppressWarnings("serial")
 public class JdotxtTaskPanel extends JPanel {
+
+	private static final Map<Priority,Color> BACKGROUND_COLOR = new EnumMap<Priority, Color>(Priority.class){{
+		put(Priority.A, new Color(230,50,0));
+		put(Priority.B, new Color(255,140,0));
+		put(Priority.C, new Color(255,200,0));
+		put(Priority.D, new Color(230,230,0));
+		put(Priority.E, new Color(100,200,100));
+		put(Priority.NONE, new Color(220,220,220));
+	}};
+
 	public final static short ADDNEW             = 0;
```

Comment:
Again -- could not find better place in the code, so put it here.

Bug:

* open todo.txt with long list, so that scroll appears on the right.
* scroll down to the bottom
* click any of the items at the very bottom
* WHOOPS -- it scrolls back up to the very top.  I would expect scroll bar to not be affected by focus change.